webServer.js

```diff
@@ -146,9 +154,29 @@ function start() {
     return fullContent;
   };
 
-  var setCache = function (res, hours) {
-    res.setHeader('Cache-Control', 'public, max-age=' + 60 * 60 * hours + ', immutable');
-    res.setHeader('Expires', new Date(Date.now() + 60 * 60 * hours * 1000).toUTCString());
+  var cacheCheck = function (req, res, hours, lastModified) {
+    res.setHeader("Cache-Control", "public, max-age=" + 60 * 60 * hours + ", immutable");
```

Comment:
Should .toString() be used on the number in this line, or is the type coercion that is currently happening fine?

Reply:
I think it's fine.